docs/UserGuide.md

```diff
@@ -723,15 +714,11 @@ from another venue, the table will automatically switch to that venue's booking.
 Alternatively, you can use the `venue view VENUE_NAME` command to switch manually.
 </div>
 
----
-
 <div markdown="span" class="alert alert-info">:information_source: **Note:**
 Before proceeding to learn more about venue commands, it would be good to have a quick read about the **venue fields**, which will
-be used in the commands.
+be used in the commands. The venue format <a href="https://ay2223s1-cs2103t-w12-3.github.io/tp/UserGuide.html#glossary-of-terms">fields</a> can be found <a href="https://ay2223s1-cs2103t-w12-3.github.io/tp/UserGuide.html#format-for-venue-fields">here</a>.
```

Comment:
"venue format fields" sounds a little strange, perhaps consider changing to just "venue fields"?